packages/admin/cms-admin/src/contentScope/ContentScopeSelect.tsx

```diff
@@ -275,7 +275,9 @@ export function ContentScopeSelect({
                                                 selected={isSelected}
                                                 sx={({ spacing }) => ({
                                                     paddingX: spacing(6),
-                                                    fontWeight: isSelected ? 600 : 250,
+                                                    "&& .MuiListItemText-primary": {
+                                                        fontWeight: isSelected ? 600 : 250,
+                                                    },
```

Comment:
Could we use slotProps for this instead? I'm not fond of specifying MUI classes here which may change without us noticing (due to being a string).